src/prefect/cli/root.py

```diff
@@ -70,7 +74,8 @@ def main(
         soft_wrap=not PREFECT_CLI_WRAP_LINES.value(),
     )
 
-    setup_logging()
+    if not PREFECT_TEST_MODE:
+        setup_logging()
```

Comment:
Otherwise the `CliRunner` will attempt to setup logging multiple times when the logging config changes